pkg/skaffold/build/gcb/types.go

```diff
@@ -55,25 +55,25 @@ const (
 	// StatusCancelled  "CANCELLED" - Build was canceled by a user.
 	StatusCancelled = "CANCELLED"
 
-	// RetryDelay is the time to wait in between polling the status of the cloud build
-	RetryDelay = 1 * time.Second
+	// PollDelay is the time to wait in between writing logs of the cloud build
+	PollDelay = 1 * time.Second
 
 	// BackoffFactor is the exponent for exponential backoff during build status polling
 	BackoffFactor = 1.5
 
 	// BackoffSteps is the number of times we increase the backoff time during exponential backoff
 	BackoffSteps = 10
 
-	// RetryTimeout is the max amount of time to retry getting the status of the build before erroring
-	RetryTimeout = 3 * time.Minute
+	// MaxRetryCount is the max number of times we retry a throttled GCB API request
+	MaxRetryCount = 20
```

Comment:
hard to do the math in my head but does 20 tries roughly equate to the 3 minute timeout we had before?